src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -395,7 +386,16 @@ def _filter_file(tarinfo):
         # otherwise the child item name will have a prefix (eg, ../) which can block unpacking.
         tar.add(source_location, arcname="", filter=_filter_file)
 
-    logger.debug("Starting to upload the archived source code from '%s'.", tar_file_path)
+    size = os.path.getsize(tar_file_path)
+    unit = 'GiB'
+    for S in ['Bytes', 'KiB', 'MiB', 'GiB']:
```

Comment:
You know, the logic will not work if the size is 1TB+. Though this won't happen, but would be good to get the code right

Reply:
Totally, not relevant to this PR though. https://github.com/Azure/azure-cli/issues/6925 feel free to assign this to me, I'll do it later when I have time.